reddit_service_activity/__init__.py

```diff
@@ -59,18 +59,45 @@ def record_activity(self, context, context_id, visitor_id):
         self.counter.record_activity(context.redis, context_id, visitor_id)
 
     def count_activity(self, context, context_id):
-        if not _ID_RE.match(context_id):
+        results = self.count_activity_multi(context, [context_id])
+        return results[context_id]
+
+    def count_activity_multi(self, context, context_ids):
```

Comment:
Explaining what the input data looks like would be nice, since `context` and `context_id` aren't super obvious.